docs/examples/shading/plot_far_shading_with_pvgis_horizon_data.py

```diff
@@ -0,0 +1,96 @@
+"""
+Far-Shading with PVGIS Horizon Data
+=========================
+
+Example of getting PVGIS horizon data, interpolating it to time-series
+solar-position data, and adjusting DNI and POA-global irradiance.
+"""
+
+# %%
+# This example shows how to use retrived horizon elevation angles with
+# corresponding horizon azimuth angles from the
+# :py:meth:`pvlib.iotools.get_pvgis_horizon` furntion.
+
+# After location information and a date range is established, solar position
+# data is queried from :py:meth:`pvlib.solar_position.get_solar_position`.
+# Horizon data is then retreived, and interpolated to the solar azimuth time
+# series data. Finally, in times when solar elevation is greater than the
+# interpolated horizon elevation angle, DNI is set to 0.
+
+import numpy as np
+import pandas as pd
+from pvlib.iotools import get_pvgis_horizon
+from pvlib.location import Location
+from pvlib.irradiance import get_total_irradiance
+
+# Golden, CO
+lat, lon = 39.76, -105.22
+tz = 'MST'
+
+# Set times in the morning of the December solstice.
+times = pd.date_range(
+    '2020-12-20 6:30', '2020-12-20 9:00', freq='1T', tz=tz, inclusive='left')
```

Comment:
```suggestion
    '2020-12-20 6:30', '2020-12-20 9:00', freq='1T', tz=tz)
```

This parameter isn't available on all pandas versions, and not really needed here, so better to not use it.